vowpalwabbit/fb_parser/src/parse_example_flatbuffer.cc

```diff
@@ -342,23 +445,13 @@ int parser::parse_namespaces(VW::workspace* all, example* ae, const Namespace* n
       // assuming the usecase that if feature names would exist, they would exist for all features in the namespace
       if (ns->feature_names()->size() != ns->feature_values()->size())
```

Comment:
The implicit assumption is that there are either hashes or names and that that must match the values collection in size.  An explicit comment here would be helpful for future us :).